readthedocs/projects/forms.py

```diff
@@ -202,6 +201,22 @@ class Meta(object):
             # 'num_major', 'num_minor', 'num_point',
         )
 
+    def __init__(self, *args, **kwargs):
+        super(ProjectAdvancedForm, self).__init__(*args, **kwargs)
+
+        default_choice = (None, '-' * 9)
```

Comment:
I Pythonic way to do this would be to use something like `empty_value`: https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.TypedChoiceField.empty_value

I'm not 100% sure that this is possible here, but I think it should be.

Reply:
We are just replacing the widget (`Select`), so it's not possible https://docs.djangoproject.com/en/2.1/ref/forms/widgets/#django.forms.Select